app/pages/about/faq.cjsx

```diff
@@ -62,7 +62,7 @@ module.exports = React.createClass
       <p>All of our images are served to the website in random order. Unfortunately, there is no good way to see the images before and after a given image.</p>
 
       <h2>How can I use WildCam Gorongosa in my classroom?</h2>
-      <p>In the spring of 2016, we will launch an educator portal that will allow students to ask ecological questions, filter trail camera data, and download the data to analyze trends. Educators can use the portal to set up their classrooms, invite students, and manage projects and data for use in class. In the meantime, we will post activities on our <a href="http://www.wildcamgorongosa.org/education">Education</a> page using smaller sets of trail camera photos and data. You can also explore a variety of educational resources about Gorongosa National Park on <a href="http://www.biointeractive.org/gorongosa">BioInteractive.org</a>.</p>
+      <p>In the spring of 2016, we will launch an educator portal that will allow students to ask ecological questions, filter trail camera data, and download the data to analyze trends. Educators can use the portal to set up their classrooms, invite students, and manage projects and data for use in class. In the meantime, we will post activities on our <a href="http://www.wildcamgorongosa.org/about/education">Education</a> page using smaller sets of trail camera photos and data. You can also explore a variety of educational resources about Gorongosa National Park on <a href="http://www.biointeractive.org/gorongosa">BioInteractive.org</a>.</p>
```

Comment:
@srallen Hi, you seem to be tagging/notifying me instead of @aliburchard (this also subscribed me to #111, #96, and #68). Remember to use the full username when @-mentioning someone! :rainbow: